### ISSUE TITLE
Pre-Build Command line Macros not working

### ISSUE THREAD

**Issue by Toasterthegamer** [SiliconStudio/xenko#218]
```[C:\Program Files (x86)\MSBuild\12.0\bin\amd64\Microsoft.Common.CurrentVersion.targets(1131,5)]: Error: The command "xcopy "*Undefined*libraries\Image" "*Undefined*Platformer\Bin\Windows-Direct3D11\Debug" /E /Y
xcopy "*Undefined*libraries\Image" "*Undefined*Platformer\Bin\Windows-Direct3D11\Release" /E /Y" exited with code 4.```

Specifically the: "$(SolutionDir)" is failing.

Note: This error is ONLY in paradox studio not in VS.

**Comment by xoofx** [comment]
Could you give us some details on how to reproduce this problem? (steps required for creating the project...etc.)

**Comment by Toasterthegamer** [comment]
Step 1: Open up the project settings and navigate to the build events page. 
Step 2: Enter in either a Pre-Build or Post-Build command that uses the '''$(SoultionDir)''' macro. The one I have gotten to error was a pre-build.  Save project once done.
Step 3: Open up the solution in paradox studio.

From there paradox studio throws an error saying the command exited with code 4 which is due to the '''$(SoultionDir)''' equaling '''*Undefined*'''

Inside of visual studio the same pre-build or post-build command will work fine as the macro above get's filled in with the solutions directory.

**Comment by xenux** [comment]
This problem should be fixed from 1.4 so I close the issue. 
If you still experiment any problem let us know. Thanks
